staging/src/k8s.io/apiextensions-apiserver/test/integration/conversion/conversion_test.go

```diff
@@ -408,6 +423,91 @@ func validateStoragePruning(t *testing.T, ctc *conversionTestContext) {
 	}
 }
 
+func validateDefaulting(t *testing.T, ctc *conversionTestContext) {
```

Comment:
@liggitt added this: defaulting should happen in the request version when entering the request pipeline, and for the storage version when reading from etcd.